101-machine-learning-compute-create-computeinstance/azuredeploy.parameters.json

```diff
@@ -3,16 +3,31 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "workspaceName": {
-      "value": "GET-PREREQ-workspaceName"
+      "value": null
```

Comment:
we need valid values so that a test can be run on the template... if this requires existing resources (e.g. the workspace, then you need to supply a prereq template for those.
See: https://github.com/Azure/azure-quickstart-templates#template-pre-requisites for how to handle prereqs

Reply:
If there are required parameters which we need to ask the user for instance AAD object id, tenant id how do we handle that?